src/components/singleProductPage/SingleProductPage.jsx

```diff
@@ -0,0 +1,79 @@
+import React from "react";
+import Slider from "react-slick";
+import "./singleProductPage.scss";
+
+export default function SingleProductPage(props) {
+  // These varaibles need to be replaced by the real data
+  const title = props.props.title;
```

Comment:
`props.props` seems wrong to me, restructure this please. also consider using destructuring on line 5 `SingleProductPage({ title, images, brand, price , ... })` to make this code much cleaner